src/components/funnels/layout/splash.tsx

```diff
@@ -6,7 +6,7 @@ export interface SplashFunnelLayoutProps {
 	readonly children: ReadonlyReactNode
 }
 
-const SplashFunnelLayout = ({ image, children }: SplashFunnelLayoutProps) => <Splash image={image}>
+export const SplashFunnelLayout = ({ image, children }: SplashFunnelLayoutProps) => <Splash image={image}>
```

Comment:
I don't understand the purpose of this change, it is already exported on line 13 and now all the imports are also changed without (to me) any clear benefit.

Reply:
When you get to the error dialog in development mode, gatsby throws an exception that this is a "mixed import".

If this is wrong, I'll be happy to roll back this change, but that fixed the problem for me that crashed the app locally when a registration submit error occurred.